src/presentation/http/router/note.test.ts

```diff
@@ -244,7 +243,128 @@ describe('Note API', () => {
         expect(response?.json().accessRights).toStrictEqual({ canEdit: true });
       });
     });
+  });
+
+  describe('PATCH note/:notePublicId ', () => {
+    test('Update note by public id with 200 status, user is creator of the note', async () => {
+      const expectedStatus = 200;
+      const userId = 2;
+      const accessToken = global.auth(userId);
+
+      const userNote = notes.find(newNote => {
+        return newNote.creator_id === userId;
+      });
+
+      const response = await global.api?.fakeRequest({
+        method: 'PATCH',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/note/${userNote!.public_id}`,
+        body: {
+          'content': { new: 'content added' },
+        },
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
```

Comment:
does response have body? if it does, you should check if returned note contains appended data

Reply:
If i'm not mistaken, definition of fastify.post in router/note.ts says, that method replies with notePublicId, so i check the returned id to be string with length > 0
![image](https://github.com/codex-team/notes.api/assets/120342275/970ae725-0089-44c0-8d2b-a6392645ed5a)

Reply:
> so i check the returned id to be string with length > 0

where is this check?

Reply:
![image](https://github.com/codex-team/notes.api/assets/120342275/fab7f2fa-0f0f-4549-ab7c-bcb0d77eb2ee)
in the test 'Post a new note without parentId passed.'

Reply:
these checks should be added to this test case as well